src/Spd.Resource.Repository/Biz/BizRepository.cs

```diff
@@ -46,23 +46,40 @@ public async Task<IEnumerable<BizResult>> QueryBizAsync(BizsQry qry, Cancellatio
 
         public async Task<BizResult?> GetBizAsync(Guid accountId, CancellationToken ct)
         {
-            IQueryable<account> accounts = _context.accounts.Expand(a => a.spd_Organization_Addresses)
+            IQueryable<account> accounts = _context.accounts
+                .Expand(a => a.spd_Organization_Addresses)
+                .Expand(a => a.spd_organization_spd_licence_soleproprietor)
                 .Where(a => a.statecode != DynamicsConstants.StateCode_Inactive)
                 .Where(a => a.accountid == accountId);
 
-            account? Biz = await accounts.FirstOrDefaultAsync(ct);
+            account? biz = await accounts.FirstOrDefaultAsync(ct);
             
-            if (Biz == null) throw new ApiException(HttpStatusCode.NotFound);
+            if (biz == null) throw new ApiException(HttpStatusCode.NotFound);
 
             List<spd_account_spd_servicetype> serviceTypes = _context.spd_account_spd_servicetypeset
-                .Where(so => so.accountid == Biz.accountid)
+                .Where(so => so.accountid == biz.accountid)
                 .ToList();
 
+            Guid? licHolderId = biz.spd_organization_spd_licence_soleproprietor
+                .FirstOrDefault(l => l.statecode == DynamicsConstants.StateCode_Active)?
+                ._spd_licenceholder_value;
+            spd_licence? licHolderContact = new();
+
+            if (licHolderId != null && licHolderId != Guid.Empty)
+                licHolderContact = _context.spd_licences
+                    .Expand(l => l.spd_LicenceHolder_contact)
+                    .Where(l => l.statecode != DynamicsConstants.StateCode_Inactive)
+                    .Where(l => l._spd_licenceholder_value == licHolderId)
+                    .FirstOrDefault();
+

```

Comment:
We do not need this. 
 biz.spd_organization_spd_licence_soleproprietor
                .FirstOrDefault(l => l.statecode == DynamicsConstants.StateCode_Active)?
                ._spd_licenceholder_value
is the response.SoleProprietorSwlContactInfo.ContactId

 biz.spd_organization_spd_licence_soleproprietor
                .FirstOrDefault(l => l.statecode == DynamicsConstants.StateCode_Active)?
                .spd_licenceId
is response.SoleProprietorSwlContactInfo.LicenceId

Reply:
Fixed.